src/main/common/maths.c

```diff
@@ -111,7 +111,7 @@ int32_t applyDeadband(int32_t value, int32_t deadband)
     return value;
 }
 
-inline int constrain(int amt, int low, int high)
```

Comment:
I've moved constrain and constrainf to the include file and re-inlined them to be properly optimized by the compiler.